lesson_25/amiyahjones/src/App.tsx

```diff
@@ -0,0 +1,25 @@
+import './App.scss';
+import {Outlet, Route, Routes} from 'react-router-dom';
+
+import {Footer} from './components/footer';
+import {Header} from './components/header';
+import { CreateProgram } from './pages/Add-program/CreateProgram';
+
+function App() {
+  return (
+    <>
+      <Header />
+      <Routes>
+        <Route path='' element={<CreateProgram/>} />
+      </Routes>
```

Comment:
Tutorials and such has thrown me off, perhaps the pace of them or just my mind overall. It's left me at a roadblock to linking my new `CreatePrograms` page to begin with as I'm lost with these files

Reply:
See note in main.tsx

Reply:
Also, don't need the `<Routes>` tag here.

Reply:
done!